json-schema-maven-plugin/pom.xml

```diff
@@ -44,13 +44,13 @@
         <dependency>
             <groupId>org.ow2.asm</groupId>
             <artifactId>asm</artifactId>
-            <version>7.1</version>
+            <version>9.0</version>
             <scope>test</scope>
         </dependency>
     </dependencies>
 
     <properties>
-        <junit-version>4.13</junit-version>
+        <junit-version>4.13.1</junit-version>
```

Comment:
There's some duplication here, specifying the property in each POM. At the very least, this property should be moved to the top-level POM. Even better, the junit dependency (and in fact, any dependency used in more than one module) should be moved to a `dependencyManagement` section in the top-level POM if we don't want it automatically included in ever module. Those that use it would then list it in their dependencies without a version, inheriting the one from the project.

Reply:
Mark moved the JUnit 4 dependency down into the individual modules because he didn't want any chance that the new integration tests would use JUnit 4 symbols. I'm happy to change this however will work. I wonder if we could just universally switch to JUnit 5 now?